### PR TITLE
Request from CTLearn to join OpenAstronomy as a sub-org

### PR DESCRIPTION
**Pull request by nietootein**
Hello OpenAstronomy!

This is CTLearn, a package to explore deep-learning methods for the analysis of data from imaging atmospheric Cherenkov telescopes (very-high-energy gamma-ray astronomy). We would be thrilled to join you as a sub-org!

Best,

CTLearn

### CODE REVIEW

File: _data/members.yaml [hunk 1]
```diff
@@ -244,6 +244,22 @@ gnuastro:
     <a href="https://www.gnu.org/software/gnuastro/manual/html_node/Tutorials.html"
     target="_blank">tutorials</a> is the best place to start using it.
 
+ctlearn:
+   name: CTLearn
+   logo: ctlearn.png   
+   url: https://github.com/ctlearn-project
+   repositories:
+     github: https://github.com/ctlearn-project/ctlearn.git
```
Comment (dpshelio) [inline _data/members.yaml]:
```suggestion
     github: ctlearn-project/ctlearn
```

Comment (nietootein) [reply _data/members.yaml]: Suggestion committed. I also added our Slack coordinates as a first contact for students. 

Thanks!

**Comment by bsipocz** [comment]
> @OpenAstronomy/committee I'm happy as it is. Any comment?

I'm 👍 in general, but it's not at all clear whether this project is open development. There is neither documentation nor a `CONTRIBUTING.md` file, in fact the word "contribute" is not present anywhere.

**Comment by mirca** [comment]
Likewise @bsipocz, I'm +1, but still concerned about the lack of documentation.

**Comment by nietootein** [comment]
Thanks so much for your support! Your concerns as to the lack of documentation and contributing guidelines are legit and we will try to address them as soon as possible.

**Pull request merged**